hrjobcontract/js/test/controllers/modal/modal-contract-new.controller.spec.js

```diff
@@ -251,6 +255,7 @@ define([
       it('calls the crmAngService with the requested url', function () {
         expect(crmAngService.loadForm).toHaveBeenCalledWith(providersPopupLists[0].popupFormUrl);
```

Comment:
You already have 3 lists so it may make sense to wrap in a cycle. This would also mean you will have to worry less if you add more lists.

```js
_.each(providersPopupLists, function (popupList) {
  expect(crmAngService.loadForm).toHaveBeenCalledWith(popupList.popupFormUrl);
});
```